src/components/screens/ScoreSetView.vue

```diff
@@ -108,14 +108,16 @@
             ref="histogram"
           />
         </div>
-        <div v-if="showHeatmap" class="mave-score-set-heatmap-pane">
+        <div v-if="showHeatmap && !isScoreSetVisualizerVisible" class="mave-score-set-heatmap-pane">
           <ScoreSetHeatmap
             :scoreSet="item"
             :scores="scores"
             :externalSelection="variantToVisualize"
+            :showProteinStructureModal="uniprotId!=null"
```

Comment:
Since for now we'll only want to show the model opening option when the clinical preview settings are enabled, let's add another computed element:

```
    protein-structure-viewer-enabled: function() {
      return uniprotId != null && this.config.CLINICAL_FEATURES_ENABLED
    },
```

If it's not already, you may also need to import the config object and add it's ref to the data:
```
import config from '@/config'
...

data: function() {
    ...

    return {
      config: config,
      ...
    }
  },

Reply:
Ok, Makse sense. I also realized this was a non-existent prop, what I actually wanted was `showProteinStructureButton`

For now, since this calc isn't likely to get more complex, I just put it directly on the prop:
```
:showProteinStructureButton="uniprotId!=null && config.CLINICAL_FEATURES_ENABLED"
```